cinema/views.py

```diff
@@ -1,45 +1,111 @@
-from rest_framework.decorators import api_view
+from rest_framework import generics, viewsets, status, mixins
+from django.shortcuts import get_object_or_404
+
 from rest_framework.response import Response
-from rest_framework import status
+from rest_framework.views import APIView
 
-from django.shortcuts import get_object_or_404
+from cinema.models import Movie, Actor, Genre, CinemaHall
+from cinema.serializers import (
+    MovieSerializer,
+    ActorSerializer,
+    GenreSerializer,
+    CinemaHallSerializer,
+)
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.all()
+    serializer_class = MovieSerializer
+
+
+class ActorList(
+    generics.ListCreateAPIView,
+):
+    queryset = Actor.objects.all()
+    serializer_class = ActorSerializer
+
+    def get(self, request, *args, **kwargs):
+        return self.list(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        return self.create(request, *args, **kwargs)
+
+
+class ActorDetail(
+    generics.GenericAPIView,
+    mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
+    mixins.DestroyModelMixin,
```

Comment:
Consider using `RetrieveUpdateDestroyAPIView` for `ActorDetail` to simplify the code, as it already provides `get`, `put`, `patch`, and `delete` methods.